web/src/components/ExpandedViewMode/ExpandedViewMode.scss

```diff
@@ -31,6 +30,17 @@
   position: fixed;
   top: 0;
   z-index: 2;
+  background: var(--bg-color-popup);
+
+  // !DARK: needs plumbing
```

Comment:
not following this comment

Reply:
im referring to the fact that the react component this css applies to does not currently use the `theme` variable from the redux state, but it needs to